### ISSUE TITLE
Clearer instructions on how to use please

### ISSUE THREAD

**Issue by benknight** [filamentgroup/fixed-sticky#37]
I've failed every time I've tried to use this plugin and I swear I'm an experienced web developer and not a complete idiot.  Results are mixed but never quite what I'm expecting.  DOM structure seems to matter a lot here.  Also the plugin doesn't seem to account for margins and paddings in some cases?

**Comment by zachleat** [comment]
I hear you, sir! I’m really glad you posted your frustrations here—I’ve made some changes to the documentation—especially around the heavy restriction of sticky elements staying inside their parent containers. Will you take a look and let me know if that’s more clear? Would love your help on improving it.

As for container padding, this is currently highlighted on Issue #35.

**Comment by benknight** [comment]
Thanks for the quick feedback.  After I wrote this I eventually figured out how the containers thing worked, which was definitely a breakthrough!  I also realized that in order for it to work on a direct descendent of the `body` tag I had to remove `height: 100%` from my CSS.  I think it's also probably good to point out that margins aren't maintained, only paddings (I think) on the fixedsticky element.
